proto/viam/app/v1/app.proto

```diff
@@ -144,20 +144,72 @@ message ListOrganizationsResponse {
   repeated Organization organizations = 1;
 }
 
+// Location
+//
+
+message LocationOrganization {
+  // Organization ID the location is shared with.
+  string organization_id = 1;
+  // Whether the organization is the primary owner or not.
+  bool primary = 2;
+}
+
+message LocationAuth {
+  // Deprecated: use secrets field.
```

Comment:
I think we should just drop this.

Reply:
Do you know of a way to delete the field and get the buf-breaking checks to pass? I tried to add `reserved`. Any other way to ignore/bypass the check since we want it deleted?

Reply:
As we discussed, adding the field back for now :)